config/autoload.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class MyAutoload
+{
+    public static function start()
+    {
+        spl_autoload_register('MyAutoload::autoload');
+    }
+    
+    public static function autoload($class)
+    {
+        if(file_exists("model/$class.php"))
+        {
+            include_once("model/$class.php");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["include_once" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)